app/models/classification.rb

```diff
@@ -31,12 +32,14 @@ def self.scope_for(action, user, opts={})
       complete.merge(created_by(user)).includes(:subjects)
     when :show
       created_by(user).includes(:subjects)
-    when :update, :destroy
+  when :update, :destroy
       incomplete_for_user(user)
     when :incomplete
       incomplete_for_user(user).includes(:subjects)
     when :project
       joins(:project).merge(Project.scope_for(:update, user)).includes(:subjects)
+    when :project_offset
+      after_id(opts[:last_id]).joins(:project).merge(Project.scope_for(:update, user))
```

Comment:
seems project and project_offset should collapse into one (project in my mind) and just merge the extra offset scope if the param is set. Sound ok? Also does the last_id param make sense without the project_id...maybe. If not then perhaps ensure the project_id is a req'd param with last_id.